portfolio/src/main/webapp/index.html

```diff
@@ -71,6 +71,11 @@
               >Interests</a
             >
           </li>
+          <li class="nav-item">
+            <a class="nav-link js-scroll-trigger" href="#comments"
+              >Comments</a
```

Comment:
I just noticed this.
I think it is better to have the ">" on the same line as the "<" to make it easier to read. e.g:
```
<a class="nav-link js-scroll-trigger" href="#comments">
    Comments
</a>
```
or if you cant, we can put the attributes into multiple lines, like so:
```
<a class="nav-link js-scroll-trigger" 
     href="#comments">
    Comments
</a>
```
I feel that this is cleaner, what do you think?

Reply:
Yes, definitely!